src/components/ui/SectionTitle/SectionTitle.js

```diff
@@ -0,0 +1,12 @@
+import { Headline } from '../Headline/Headline';
+import { cn } from '../../../lib/utils';
+
+export const SectionTitle = ({ title, className, type }) => {
+  return (
+    <Headline
+      className={cn(`text-white w-full border-b border-white tracking-widest`, className)}
+      type={type}>
```

Comment:
type should be already preselected, if you expose it like this you didn't get much of this component, because you could do it directly on place where you need this. But instead if you already preselect type you made this component and nothing can't be changed, so check design which type fits best and put it here